### DISCUSSION TITLE
How I change the npmrc to yarnrc.yml in yarn v3.6

### DISCUSSION THREAD

**Discussion by LauGaHo** [yarnpkg/berry#5522] [Q&A]
the `.npmrc` as follow:
<img width="1068" alt="iShot_2023-06-24_18 26 54" src="https://github.com/yarnpkg/yarn/assets/52206004/b898766c-4704-42ba-875b-deeb12063b0f">
so how I write in the `yarnrc.yml`, I try much times, and I has failed. pls help me

**Comment by merceyz** [comment]
All the configuration options are listed here https://yarnpkg.com/configuration/yarnrc, the ones relevant for your screenshot are
- https://yarnpkg.com/configuration/yarnrc#npmAlwaysAuth
- https://yarnpkg.com/configuration/yarnrc#caFilePath
- https://yarnpkg.com/configuration/yarnrc#enableStrictSsl
